### ISSUE TITLE
[be/order-management] order service needs method to update orders of matches

### ISSUE THREAD

**Issue by tunggad** [beesEX/be#9]
Summary:
When trading engine matches orders, it will emit OrderbookEvent events, these events will then processed by the trade execution service of settlement module to execute the trade, after crediting and debiting the participating currency accounts accordingly, it also needs to update the relating orders of each match. For that purpose order service should provide a appropriate function, which receives the reason-object and match-object of OrderbookEvent for each match as input and updates the order records in DB accordingly. This function will be then called by the trade execution service.

This task is required by #7 and has to be done first.

TODO:
Creates a function called `updateOrdersbyMatch(reasonObj, matchObj)` of order.service.js, it should updates **filledQuantity**, **status** and **lastUpdatedAt** fields of the two matching orders. The function should return true if the update was successful, else false.

**Comment by tunggad** [comment]
@hovietanh1991: this task needs to be adapted because it was described to work together with my original solution, which was not choosen, we have choosen the solution of Son instead, so this needs to be adapted to that, i will do it ASAP, but not now, im busy on other tasks.

**Comment by tunggad** [comment]
@hovietanh1991: this task has been aligned with #13 and is now ready for implement

**Comment by tunggad** [comment]
demo worked!
